bulldozer/merge.go

```diff
@@ -152,7 +152,7 @@ func (m *PushRestrictionMerger) DeleteHead(ctx context.Context, pullCtx pull.Con
 
 // MergePR merges a pull request if all conditions are met. It logs any errors
 // that it encounters.
-func MergePR(ctx context.Context, pullCtx pull.Context, merger Merger, mergeConfig MergeConfig) {
+func MergePR(ctx context.Context, pullCtx pull.Context, merger Merger, mergeConfig MergeConfig) string {
```

Comment:
While `string` is functional here, perhaps it might be better to return an `error` now that something checks the return. That could centralize some logging and a custom error type could be used to signal errors that should be reported as comments.